api/v1beta1/events.go

```diff
@@ -10,83 +10,91 @@ import (
 )
 
 const (
-	EventOnTargetTemplate     string = "Failing probably caused by disruption %s: "
-	SourceDisruptionComponent string = "disruption-controller"
+	EventOnTargetTemplate         string = "Failing probably caused by disruption %s: "
+	SourceDisruptionComponent     string = "disruption-controller"
+	SourceDisruptionCronComponent string = "disruptioncron-controller"
 )
 
-type DisruptionEventCategory string
+type EventCategory string
 
 const (
-	// Event only attached to a target
-	TargetEvent DisruptionEventCategory = "TargetEvent"
-	// Event only attached to the disruption
-	DisruptEvent DisruptionEventCategory = "DisruptionEvent"
-	// Event only attached to a chaos pod
-	ChaosPodEvent DisruptionEventCategory = "ChaosPodEvent"
+	// TargetEvent only attached to a target
+	TargetEvent EventCategory = "TargetEvent"
+	// DisruptEvent only attached to the disruption
+	DisruptEvent EventCategory = "Event"
```

Comment:
Shouldn't this still be DisruptionEvent, if the event below is DisruptionCronEvent?